app/code/Magento/Theme/Block/Html/Title.php

```diff
@@ -19,13 +22,39 @@
  */
 class Title extends Template
 {
+    /**
+     * Config path to 'Translate Title' header settings
+     */
+    private const XML_PATH_HEADER_TRANSLATE_TITLE = 'design/header/translate_title';
+
+    /**
+     * @var ScopeConfigInterface
+     */
+    private $scopeConfig;
+
     /**
      * Own page title to display on the page
      *
      * @var string
      */
     protected $pageTitle;
 
+    /**
+     * Constructor
+     *
+     * @param Template\Context $context
+     * @param array $data
+     * @param ScopeConfigInterface|null $scopeConfig
+     */
+    public function __construct(
+        Template\Context $context,
+        array $data = [],
+        ScopeConfigInterface $scopeConfig = null
+    ) {
+        parent::__construct($context, $data);
+        $this->scopeConfig = $scopeConfig ?: ObjectManager::getInstance()->get(ScopeConfigInterface::class);
```

Comment:
Before we used such constructions to keep backward compatibility.
Currently we are delivering code in Magento 2.4.
So can you please, implement this without using of ObjectManager directly

Reply:
@nuzil done